core/src/main/java/bio/terra/pearl/core/service/notification/email/EnrolleeEmailService.java

```diff
@@ -131,6 +134,28 @@ protected Mail buildEmail(NotificationContextInfo contextInfo, EnrolleeContext r
             // if this portal environment hasn't been configured with a specific email, just send from the support address
             fromAddress = routingPaths.getSupportEmailAddress();
         }
+
+        if (contextInfo.study() != null) {
+            // makes sure study.name returns the name of the study in the preferred language
+            languageTextService
+                    .findBySiteContentLanguageAndKey(
+                            contextInfo.portalEnv().getSiteContentId(),
+                            preferredLanguage,
+                            "study:" + contextInfo.portal().getShortcode() + "." + contextInfo.study().getShortcode())
```

Comment:
let's make this a method in LanguageTextService - `findTranslatedStudyName(portalShortcode, studyShortcode)` so the syntax is in a more findable place